libs/sdk-ui-kit/styles/scss/Form/_variables.scss

```diff
@@ -1,9 +1,2 @@
 // (C) 2020 GoodData Corporation
 @import "../variables";
-
-// CSS variables
-$gd-input-focus-color: var(--gd-palette-primary-base, $gd-color-highlight);
-$gd-searchfield-icon-color: var(--gd-palette-primary-base, $gd-color-highlight);
-$gd-searchfield-clear-color: var(--gd-palette-primary-base, $gd-color-highlight);
-$gd-input-checkbox-label-color: var(--gd-palette-primary-base, $gd-color-highlight);
-$gd-input-checkbox-toggle-color: var(--gd-palette-primary-base, $gd-color-highlight);
```

Comment:
I am just wondering why are you remove these variables?

Reply:
because I have introduced them few days ago and as they have all the same value I rather use $gd-palette-primary-base: var(--gd-palette-primary-base, $gd-color-highlight); to substitute them all